docs/dataframe_supported.rst

```diff
@@ -0,0 +1,458 @@
+| ``bfill``                 | Y                               |                                                    |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``blocks``                | D                               | Defaults to pandas                                 |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``bool``                  | Y                               |                                                    |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``boxplot``               | Y                               |                                                    |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``clip``                  | Y                               |                                                    |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``clip_lower``            | Y                               |                                                    |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``clip_upper``            | Y                               |                                                    |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``combine``               | D                               | Defaults to pandas                                 |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``combine_first``         | D                               | Defaults to pandas                                 |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``compound``              | D                               | Defaults to pandas                                 |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``consolidate``           | D                               | Defaults to pandas                                 |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``convert_objects``       | D                               | Defaults to pandas                                 |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``copy``                  | Y                               |                                                    |
```

Comment:
Do we do deep and shallow copies properly?

Reply:
Yes, we do now as of 0.5.1